### PR TITLE
feat(files): added getFileUrl errors

### PR DESCRIPTION
**Pull request by deminnsk**

### CODE REVIEW

File: proto/media_and_files.proto [hunk 1]
```diff
@@ -101,6 +101,14 @@ message FileUrlDescription {
     repeated HTTPHeader unsigned_url_headers = 5 [(dlg).log="hidden"];
 }
 
+// Error, that describes failure of concrete file
```
Comment (ostronom) [inline proto/media_and_files.proto]: Concrete is this -- https://www.google.com/search?q=concrete

"of particular file"

File: proto/media_and_files.proto [hunk 2]
```diff
@@ -101,6 +101,14 @@ message FileUrlDescription {
...
+message FileUrlError {
+    int64 fileId = 1 [(dlg).log="visible"]; /// the id of file 
+    string tag = 2 [(dlg).log="visible"]; /// Error tag: FILE_STORAGE_ERROR - unknown error storage, FILE_STORAGE_FORBIDDEN - has no access to this file storage
```
Comment (ostronom) [inline proto/media_and_files.proto]:
```suggestion
message Error {
  enum ErrorTag {
     FILEURLERROR_UNKNOWN = 0;
     FILEURLERROR_STORAGE_ERROR = 1;
     ...
  }
  ErrorTag tag = 1;
  string reason = 2;
}
```

File: proto/media_and_files.proto [hunk 3]
```diff
@@ -113,6 +121,7 @@ message ResponseGetFileUrl {
     int32 timeout = 2;
     google.protobuf.StringValue unsigned_url = 3;
     repeated HTTPHeader unsigned_url_headers = 4;
+    FileUrlError error = 5;
```
Comment (ostronom) [inline proto/media_and_files.proto]: Errors in request? :)

File: proto/media_and_files.proto [hunk 4]
```diff
@@ -101,6 +101,14 @@ message FileUrlDescription {
     repeated HTTPHeader unsigned_url_headers = 5 [(dlg).log="hidden"];
 }
 
+// Error, that describes failure of concrete file
+message FileUrlError {
+    int64 fileId = 1 [(dlg).log="visible"]; /// the id of file
```
Comment (ostronom) [inline proto/media_and_files.proto]: Naming convention violation. We use snake_case in proto files.

File: proto/media_and_files.proto (hunk 1 above)
Comment (vagola) [reply proto/media_and_files.proto]: of specific file

File: proto/media_and_files.proto (hunk 3 above)
Comment (vagola) [reply proto/media_and_files.proto]: This change are for `ResponseGetFileUrl` message. I almost fell for it too. Just click the big blue thing to expand code.

**Pull request merged**